src/openrct2-ui/windows/Cheats.cpp

```diff
@@ -1184,30 +1178,30 @@ static void window_cheats_paint(rct_window *w, rct_drawpixelinfo *dpi)
             colour |= COLOUR_FLAG_INSET;
         }
         sint32 actual_month = month_spinner_value - 1;
-        gfx_draw_string_left(dpi, STR_BOTTOM_TOOLBAR_CASH,          gCommonFormatArgs, colour, w->x + XPL(0) + TXTO, w->y + YPL(2) + TXTO);
-        gfx_draw_string_left(dpi, STR_YEAR,                         nullptr, COLOUR_BLACK, w->x + XPL(0) + TXTO, w->y + YPL(7) + TXTO);
-        gfx_draw_string_left(dpi, STR_MONTH,                        nullptr, COLOUR_BLACK, w->x + XPL(0) + TXTO, w->y + YPL(8) + TXTO);
-        gfx_draw_string_left(dpi, STR_DAY,                          nullptr, COLOUR_BLACK, w->x + XPL(0) + TXTO, w->y + YPL(9) + TXTO);
-        gfx_draw_string_right(dpi, STR_FORMAT_INTEGER,              &year_spinner_value, w->colours[1], w->x + WPL(1) - 10 - TXTO, w->y + YPL(7) + TXTO);
-        gfx_draw_string_right(dpi, STR_FORMAT_MONTH,                &actual_month, w->colours[1], w->x + WPL(1) - 10 - TXTO, w->y + YPL(8) + TXTO);
-        gfx_draw_string_right(dpi, STR_FORMAT_INTEGER,              &day_spinner_value, w->colours[1], w->x + WPL(1) - 10 - TXTO, w->y + YPL(9) + TXTO);
+        gfx_draw_string_left(dpi, STR_BOTTOM_TOOLBAR_CASH, gCommonFormatArgs, colour, w->x + 11 + 3, w->y + 90 + 3);
+        gfx_draw_string_left(dpi, STR_YEAR, nullptr, COLOUR_BLACK, w->x + 11 + 3, w->y + 195 + 3);
+        gfx_draw_string_left(dpi, STR_MONTH, nullptr, COLOUR_BLACK, w->x + 11 + 3, w->y + 216 + 3);
+        gfx_draw_string_left(dpi, STR_DAY, nullptr, COLOUR_BLACK, w->x + 11 + 3, w->y + 237 + 3);
+        gfx_draw_string_right(dpi, STR_FORMAT_INTEGER, &year_spinner_value, w->colours[1], w->x + 226 - 3, w->y + 195 + 3);
+        gfx_draw_string_right(dpi, STR_FORMAT_MONTH, &actual_month, w->colours[1], w->x + 226 - 3, w->y + 216 + 3);
+        gfx_draw_string_right(dpi, STR_FORMAT_INTEGER, &day_spinner_value, w->colours[1], w->x + 226 - 3, w->y + 237 + 3);
```

Comment:
I see many `11 + 3` here, why are they separate? Marging and padding?

Reply:
Indeed — the result of manually expanding macros. Looks like I've forgotten to combine these values.